spec/features/user_can_view_all_horses_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../spec_helper'
+
+RSpec.describe "when user visits horses path" do
+  it "they view all horses" do
+    Jockey.create(name: "Joey")
+    Breed.create(name: "Palomino")
+
+    horse1 =  Horse.create(name: "Penelope", age: 29, total_winnings: 34000, breed_id: 1, jockey_id: 1)
+    horse2 =  Horse.create(name: "Pepe", age: 39, total_winnings: 1000000, breed_id: 1, jockey_id: 1)
```

Comment:
Doesn't look like you actually used this horse in your expectations. Maybe you didn't need to create it? Two horses probably gives you as much information as you need on that page.